W-11092378 ACM 2-7 doc changes for permission sets (#272)
mulesoft/docs-api-community-manager: modules/ROOT/pages/create-community.adoc

* Configure the member and guest user settings automatically using the default settings specified by the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
* Configure the member and guest user settings manually using custom settings applied in the following steps.

> API Community Manager uses the following permission sets to define the visibility and access to external objects and Apex classes. 

Reviewer: Do you need to say "the visibility and"? I would think that access implies visibility, but I don't know how the permission sets work.

If you leave in "visibility," the sentence needs some rewording because "visibility to external objects" (implied by the wording) is not grammatical. Maybe:

API Community Manager uses the following permission sets to define the visibility of external objects and Apex classes and access to these entities.

Author: Your rewrite makes sense. I will make this change.

* ACM Admin User
* ACM Guest User
* ACM Member User

For more information about using permission sets, see https://help.salesforce.com/s/articleView?id=sf.perm_sets_overview.htm&type=5[Permission Sets].

=== Configure Member User Permission Set 

The permission for the member user must be assigned to the *ACM Member User* permission set so your members can access APIs, create client applications, and manage their details. 

As part of the API Community Manager package, the member user is assigned automatically. After the package is installed, a trigger searches for users with a Customer Community Profile and assigns the member users to the *ACM Member User* permission set.  

> If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.

Reviewer: ```suggestion
To create a new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```
```suggestion
If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```

> If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.

Reviewer: ```suggestion
To create a new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```
```suggestion
If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```

> If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.

Reviewer: ```suggestion
If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```
```suggestion
To create a new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```

Reviewer: Ignore the first of the two comments. I don't know why git is doubling my comments. The second comment is the correct one.

Reviewer: Ignore the first of the two comments. I don't know why git made two comments instead of one. The second comment is the correct one.

> If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.

Reviewer: ```suggestion
To create a new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```

Author: fixed

* ACM Admin User
* ACM Guest User
* ACM Member User

The guest user cannot create client applications. 

For more information about using permission sets, see https://help.salesforce.com/s/articleView?id=sf.perm_sets_overview.htm&type=5[Permission Sets].

=== Configure Member User Permission Set 

The permission for the member user must be assigned to the *ACM Member User* permission set so your members can access APIs, create client applications, and manage their details. 

As part of the API Community Manager package, the member user is assigned automatically. After the package is installed, a trigger searches for users with a Customer Community Profile and assigns the member users to the *ACM Member User* permission set.  

If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.

To manually assign a new user to the permission set for access to the external objects and Apex classes:
. Go to the member profile and search for the assigned user.
. Assign the *ACM Guest User* to the *Community Guest User*.

[[guest-user-perms]]
=== Configure Guest User Permission Set

> The permission for the guest user must be assigned to the *ACM Guest User* permission set so your guests can access APIs, create client applications, and manage their details. 

Reviewer 2: ```suggestion
The permission for the guest user must be assigned to the *ACM Guest User* permission set so your guests can access APIs information. 
```

Author: fixed
